### ISSUE TITLE
lager_handler_watcher test is failing in TravisCI [JIRA: RIAK-2553]

### ISSUE THREAD

**Issue by mrallen1** [erlang-lager/lager#356]
I noticed on #355 that all of the Travis tests failed because of this:

```
module 'lager_handler_watcher'
  lager_handler_watcher: reinstall_on_initial_failure_test_...[6.010 s] ok
  lager_handler_watcher: reinstall_on_runtime_failure_test_...*failed*
in function lager_handler_watcher:'-reinstall_on_runtime_failure_test_/0-fun-4-'/0 (src/lager_handler_watcher.erl, line 183)
**error:{badmatch,undefined}
```

🙍

**Comment by aidecoe** [comment]
I am running tests locally. lager_handler_watcher fails:

```
  lager_handler_watcher: reinstall_on_runtime_failure_test_...*failed*
in function lager_handler_watcher:'-reinstall_on_runtime_failure_test_/0-fun-1-'/2 (src/lager_handler_watcher.erl, line 184)
in call from lager_handler_watcher:'-reinstall_on_runtime_failure_test_/0-fun-4-'/0 (src/lager_handler_watcher.erl, line 184)
**error:{assertEqual,[{module,lager_handler_watcher},
              {line,184},
              {expression,"lists : flatten ( Msg )"},
              {expected,"Lager event handler lager_crash_backend exited with reason crash"},
              {value,"Lager failed to install handler lager_crash_backend into lager_event, retrying later : {error,\n
                                     crashed}"}]}
  output:<<"">>
```

I have tried different versions: 3.0.2, 3.0.3, 3.2.0.

**Comment by mrallen1** [comment]
@macintux merged #360 a couple days ago. That should resolve this issue permanently.

**Comment by aidecoe** [comment]
Thanks! Could you make a bug-fix release, please?

**Comment by mrallen1** [comment]
This was merged and tagged in release 3.2.1, so I'm going to close this now.
